index.cjs

```diff
@@ -203,6 +232,75 @@ program.command('delete-app-component')
         options.teamName, options.projectName, options.appName, execCmd);
 });
 
+program.command('get-apk-count-component')
+  .description('helps you to get apps iOS/android to component service')
+  .option('--endPoint <string>', 'Qyrus endpoint provided by Qyrus admin')
+  .option('-u, --username <string>', 'Qyrus admin provided email')
+  .option('-p, --passcode <string>', 'Qyrus admin provided passcode in base64 format')
+  .option('--teamName <string>', 'Team name you can find by logging into Qyrus app.')
+  .option('--projectName <string>', 'Project name you can find by logging into Qyrus app.')
+  .action((options) => {
+    appCountMobilityUtil.trigger(options.endPoint, options.username, options.passcode, 
+        options.teamName, options.projectName);
+});
+
+program.command('get-apk-component')
+  .description('helps you to get apps iOS/android to mobility service')
```

Comment:
say - "helps you to get uploaded app names from mobility services for iOS & Android based projects"